Projects/UOContent/Mobiles/Monsters/Misc/Melee/PlagueBeast.cs

```diff
@@ -1,13 +1,29 @@
+using ModernUO.Serialization;
 using System;
 using Server.Engines.Plants;
 using Server.Items;
 using Server.Network;
 
 namespace Server.Mobiles
 {
-    public class PlagueBeast : BaseCreature, IDevourer
+    [SerializationGenerator(0, false)]
+    public partial class PlagueBeast : BaseCreature, IDevourer
     {
-        private int m_DevourGoal;
+        [CommandProperty(AccessLevel.GameMaster)]
+        [SerializableProperty(1)]
+        public int TotalDevoured { get; set; }
```

Comment:
Change all of these to fields instead.

Reply:
```cs
        [SerializableField(0, setter: "private")]
        private bool _hasMetalChest;

        [SerializableField(1)]
        private int _totalDevoured;

        [SerializableField(2)]
        private int _devourGoal;

        [CommandProperty(AccessLevel.GameMaster)]
        public int TotalDevoured { get { return _totalDevoured; } set { _totalDevoured = value; } }

        [CommandProperty(AccessLevel.GameMaster)]
        public bool HasMetalChest { get; private set; } // can i keep the code like this on TotalDevoured Getter And Setter?
        ```

Reply:
The point of SerializedField is to generate the property, so delete the properties for those fields.

Reply:
do i need to change CommandProperty to SerializedCommandProperty?
in this way?

```cs
        [SerializedCommandProperty(AccessLevel.GameMaster)]
        [SerializableField(2)]
        private int _devourGoal;
```

Reply:
Yes

Reply:
ok but on this case?

```cs
 [SerializedCommandProperty(AccessLevel.GameMaster)]
        [SerializableField(2)]
        private int _devourGoal;

        [CommandProperty(AccessLevel.GameMaster)]
        public int DevourGoal
        {
            get => IsParagon ? _devourGoal + 25 : _devourGoal;
            set => _devourGoal = value;
        }
```

Reply:
```cs
        [CommandProperty(AccessLevel.GameMaster)]
        [SerializableProperty(2)]
        public int DevourGoal
        {
            get => IsParagon ? _devourGoal + 25 : _devourGoal;
            set => _devourGoal = value;
        }
```